### DISCUSSION TITLE
Show full content of tooltips on the first load?

### DISCUSSION THREAD

**Discussion by durianwaffle** [eclipse-langium/langium#1702] [Q&A] [answered]
Hi Langium community,

I am wondering how can I display the full content of tooltips on the first load? When I first expand the variable tooltip, the entire content isn’t fully visible and requires scrolling. However, on the second expansion, it shows completely. Is it possible to have the full content displayed on the first expansion as well?

First load: the entire content isn’t fully visible and requires scrolling
![image](https://github.com/user-attachments/assets/dab4505d-2f28-47d5-bb9f-83abd0f0a28a)
Second expansion, it shows completely
![image](https://github.com/user-attachments/assets/7a38c8f2-3606-4259-8a8f-ee5bee90c6bf)

**Comment by msujew** [comment, marked as answer]
Hey @durianwaffle,

I assume this is a limitation of VS Code. I don't think there's a lot we can do about this as a language server.

**Reply by durianwaffle** [reply]
Okay thank you @msujew
